plugins/package-managers/spm/src/main/kotlin/Spm.kt

```diff
@@ -83,7 +83,7 @@ class Spm(
         requireLockfile(definitionFile.parentFile) { definitionFile.name != PACKAGE_SWIFT_NAME }
 
         return when (definitionFile.name) {
-            PACKAGE_SWIFT_NAME -> resolveLibraryDependencies(definitionFile)
+            PACKAGE_SWIFT_NAME -> resolveDefinitionFileDependencies(definitionFile)
```

Comment:
Some comment as before: This just giving the function and parameter different names.